roland_cutstudio.py

```diff
@@ -42,6 +41,8 @@
 import shutil
 import numpy
 from functools import reduce
+import inkex
```

Comment:
I'd suggest not using `inkex` because then there is no easy way to test the script outside Inkscape. `print("...", file=sys.stderr)` also works.